### PR TITLE
ARROW-18373: Fix component drop-down, add license text

### PR DESCRIPTION
**Pull request by toddfarmer**
https://github.com/toddfarmer/test-arrow-config/issues/new/choose has updated templates deployed for easier review.

### CODE REVIEW

**Review by assignUser — APPROVED** [review]
:tada:

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 1]
```diff
@@ -23,7 +40,6 @@ body:
         - C++
         - C++ - Gandiva
         - C++ - Plasma
-        - Compute IR
         - Continuous Integration
         - Dart
```
Comment (jorisvandenbossche) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:44]: What is "Dart"?

**Review by jorisvandenbossche — COMMENTED** [review]
Is it possible to turn the selection of component(s) into labels instead of a text section in the final issue body?

File: .github/ISSUE_TEMPLATE/bug_report.yaml (hunk 1 above)
Comment (pitrou) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:44]: Probably https://dart.dev/

Comment (jorisvandenbossche) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:44]: We have a single, closed JIRA with this (https://issues.apache.org/jira/browse/ARROW-2631), so probably not worth to have a component for

**Comment by assignUser** [comment]
> Is it possible to turn the selection of component(s) into labels instead of a text section in the final issue body?

Not automatically sadly (hopefully in the future though), we need to add a workflow that parses the issue body and adds the labels.

**Comment by toddfarmer** [comment]
> > Is it possible to turn the selection of component(s) into labels instead of a text section in the final issue body?
> 
> Not automatically sadly (hopefully in the future though), we need to add a workflow that parses the issue body and adds the labels.

FYI, the [project plan document](https://docs.google.com/document/d/1UaSJs-oyuq8QvlUPoQ9GeiwP19LK5ZzF_5-HLfHDCIg/edit?usp=sharing) tracks this and related identified tasks.

**Review by jorisvandenbossche — APPROVED** [review]

**Pull request merged**